docs/enterprise/concepts.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Concepts
+sidebarDepth: 2
+description: Learn how the Pomerium Enterprise Console works.
+---
+
+# Concepts
+
+## Namespaces
+
+In the Pomerium Enterprise Console, a **Namespace** is a cornerstone organization unit. They are container objects, that behave similar to a unix directory structure.
+
+In each Namespace, you can define an organizational unit of users and groups with fine-grained access management. This enables teams to self-service the routes and policies pertinent to them. Namespaces can optionally inherit from their parent units.
+
+Namespaces enable:
+
+- Self-Service.
+- Hierarchical policy enforcement (both enforced, and optional),
+- Policy organization.
+- RBAC for the Enterprise Console itself.
+
+Each of these sub-concepts are related and build on each other to form a unified security model.
+
+### Self-Service Capabilities
```

Comment:
I feel like a concrete example of how self-service could be used, and benefit folks would be helpful here.

Reply:
We have an example that covers self-service in the next section. I'm not yet sure if the example should be moved to this section, or just referenced (with or without link) here.

Reply:
I think good as is given proximity.